oauthproxy.go

```
@@ -116,6 +120,13 @@ func NewOauthProxy(opts *Options, validator func(string) bool) *OauthProxy {
 
 	log.Printf("Cookie settings: secure (https):%v httponly:%v expiry:%s domain:%s", opts.CookieSecure, opts.CookieHttpOnly, opts.CookieExpire, domain)
 
+	aes_cipher, err := aes.NewCipher([]byte(opts.CookieSecret))
```

I'd rather guard this to just when passAccessToken is enabled, and fail hard (`log.Fatal` is probably easiest) if we can't create the cipher instead of falling back to disabled. I think this log should hint at the extra requirements for cookie secret.